test/scripts/initialize_eks_cluster.bash

```diff
@@ -37,4 +39,8 @@ eksctl create iamserviceaccount \
      --approve \
      --region $AWS_REGION
 
-REGISTRY=$ECR_REGISTRY_URL make e2e-helm-deploy
+if [ -z "$RELEASE" ]; then
+    REGISTRY=$ECR_REGISTRY_URL make e2e-helm-deploy
+else
+    make e2e-helm-deploy
```

Comment:
This would be `e2e-helm-deploy-release`?

Reply:
updated to remove installing the driver entirely from this when `RELEASE=true` to be more consistent with the other steps in kubernetes/test-infra#22908